libpromises/mod_custom.c

```diff
@@ -839,7 +839,7 @@ static bool PromiseModule_Validate(PromiseModule *module, const EvalContext *ctx
         const char *const filename =
             pp->parent_section->parent_bundle->source_path;
         const size_t line = pp->offset.line;
-        Log(LOG_LEVEL_ERR,
+        Log(LOG_LEVEL_VERBOSE,
```

Comment:
I'm not sure we should rely on the modules providing proper error logging. I'd prefer some noise over a potential lack of very important information.

Reply:
Well, with the current implementation it is impossible to log things in a nice way. I can submit a follow-up PR to check that the module sent an error.

Reply:
https://github.com/cfengine/core/pull/4706